### PR TITLE
Update composer specifics

### PR DESCRIPTION
**Pull request by samuel20miglia**
Hi everyone,
on composer i have add/update:

- homepage
- support with his sub keys :
               "docs": "https://docs.zendframework.com/zend-expressive/",
		"issues": "https://github.com/zendframework/zend-expressive/issues",
		"source": "https://github.com/zendframework/zend-expressive",
		"irc": "http://webchat.freenode.net/?channels=zftalk",
		"forum": "https://discourse.zendframework.com/"

Thank you!

### CODE REVIEW

**Comment by samuel20miglia** [comment]
I have no idea why when i push to the branch the file rewrite tabs and spaces....
i am really desolate.
screenshot from 2017-06-01 22-10-58

File: composer.json [hunk 1]
```diff
@@ -1,79 +1,87 @@
...
+		"cs-fix": "phpcbf",
+		"test": "phpunit --colors=always",
+		"test-coverage": "phpunit --coverage-clover clover.xml",
+		"license-check": "vendor/bin/docheader check src/ test/"
+	},
+	"support": {
+		"docs": "https://docs.zendframework.com/zend-expressive/",
+		"issues": "https://github.com/zendframework/zend-expressive/issues",
+		"source": "https://github.com/zendframework/zend-expressive",
+		"irc": "http://webchat.freenode.net/?channels=zftalk",
```
Comment (weierophinney) [inline composer.json]: Remove this, and instead have the following entry:

```json
"slack": "https://zendframework-slack.herokuapp.com",
```

File: composer.json [hunk 2]
```diff
@@ -1,79 +1,87 @@
...
+		"forum": "https://discourse.zendframework.com/"
```
Comment (weierophinney) [inline composer.json]: Link this to the more specific Expressive category: https://discourse.zendframework.com/c/questions/expressive

File: composer.json (hunk 1 above)
Comment (samuel20miglia) [inline composer.json]: just fixed

File: composer.json (hunk 2 above)
Comment (samuel20miglia) [inline composer.json]: also this, just fixed

File: composer.json [hunk 3]
```diff
@@ -1,79 +1,87 @@
...
     "suggest": {
-        "filp/whoops": "^2.1 to use the Whoops error handler",
-        "zendframework/zend-expressive-helpers": "^3.0 for its UrlHelper, ServerUrlHelper, and BodyParseMiddleware",
-        "aura/di": "^3.2 to make use of Aura.Di dependency injection container",
-        "xtreamwayz/pimple-container-interop": "^1.0 to use Pimple for dependency injection",
-        "zendframework/zend-expressive-tooling": "For migration and development tools; require it with the --dev flag",
-        "zendframework/zend-servicemanager": "^3.3 to use zend-servicemanager for dependency injection"
+	"filp/whoops": "^2.1 to use the Whoops error handler",
+	"zendframework/zend-expressive-helpers": "^3.0 for its UrlHelper, ServerUrlHelper, and BodyParseMiddleware",
+	"aura/di" : "^3.2 to make use of Aura.Di dependency injection container",
```
Comment (danizord) [inline composer.json]: Remove white space before colon

Comment (samuel20miglia) [inline composer.json]: fixed

File: composer.json [hunk 4]
```diff
@@ -1,79 +1,87 @@
...
     "description": "PSR-7 Middleware Microframework based on Stratigility",
+    "homepage": "https://docs.zendframework.com/zend-expressive/",
     "type": "library",
     "license": "BSD-3-Clause",
     "keywords": [
-        "http",
-        "middleware",
-        "psr",
-        "psr-7"
+	"http",
```
Comment (weierophinney) [inline composer.json]: Please use spaces for indentation, not tabs.

File: composer.json [hunk 5]
```diff
@@ -1,79 +1,87 @@
...
+	"middleware",
+	"psr",
+	"psr-7"
```
Comment (weierophinney) [inline composer.json]: Let's also add `psr-11` to this list, since we target that.

**Comment by weierophinney** [comment]
Thanks, @samuel20miglia. Cherry-picked to master, and also merged to develop.

**Comment by samuel20miglia** [comment]
And only my pleasure, I hope in a long collaboration

**Pull request closed without merging**